crates/autopilot/src/solvable_orders.rs

```diff
@@ -244,9 +245,21 @@ impl SolvableOrdersCache {
             prices,
         };
         let removed = counter.record(&auction.orders);
-        order_events.extend(removed.into_iter().map(|o| (o, OrderEventLabel::Filtered)));
-
-        self.database.store_order_events(&order_events).await;
+        filtered_order_events.extend(removed);
+
+        // spawning a background task since `order_events` table insert operation takes
+        // a while and the result is ignored.
+        let db = self.database.clone();
+        tokio::spawn(async move {
+            db.store_invalid_order_events(&invalid_order_uids).await;
+            db.store_order_events(
```

Comment:
Shouldn't we also store just state changes for `Filtered` orders? What's the point of seeing multiple Filtered events in a row? This would make the insert code more consistent.

Reply:
Could not analyze prod data since SQL queries are a bit complex to find occurrences of subsequent `filtered` labels within a single order. Staging data was used to calculate the ratio of subsequent events with the same label, while prod data was used to determine the ratio between events with different labels:
1. There are around 45% of `invalid` events where ~99% are subsequent, meaning the table size is expected to be decreased by ~45%(14GB).
2. Around 22% of `filtered` events will remain in the table and ~70% are subsequent. So, approximately ~15% could be additionally removed(2.5GB) from the remaining data.

In summary: 45% or reduction with `invalid` retreatment only and 55% including `filtered`.